src/Imath/half.h

```diff
@@ -180,8 +171,6 @@ imath_half_to_float (imath_half_bits_t h)
     // (1.06 vs 1.08 ns/call) to avoid the constants and just do 4
     // shifts.
     //
-    //uint32_t hexpmant = (h & ~0x8000) << 13;
-    //v.i = (h & 0x8000) << 16;
     uint32_t hexpmant = ( (uint32_t)(h) << 17 ) >> 4;
     v.i = ((uint32_t)(h >> 15)) << 31;
 
```

Comment:
Should we also remove the definition of long_convert from the header of half.h? I don't see that here in the diff view...

Reply:
I missed it in the first commit but submitted a followup.